client/src/main/scala/drt/client/components/ContactPage.scala

```diff
@@ -158,7 +158,7 @@ object FeedBackComponent {
                     ),
                     MuiGrid(item = true, xs = 12)(
                       MuiTypography(variant = "h7", sx = SxProps(Map("color" -> "white", "padding" -> "2px 0")))(
-                        "Complete a short survey (approx. 2 minutes)"
+                        "Complete a short survey ( takes 2 minutes to complete )"
```

Comment:
Remove the extra white space between the brackets and their contents `(takes 2 minutes to complete)`